### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 0xYujan**
## What does this PR do?
Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
please combine with #8803, or move the CONTRIBUTING translation to this PR, as we need to consider them together

File: docs/README.md [hunk 1]
```diff
@@ -69,7 +69,8 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
   - [How-to](HOWTO-ko.md)
 - Malayalam / മലയാളം
 - Nepali / नेपाली
-  - [आचार संहिता](CODE_OF_CONDUCT-np.md) 
+  - [आचार संहिता](CODE_OF_CONDUCT-np.md)
+  - [Contributing](CONTRIBUTING-np.md)
```
Comment (shrn01) [inline docs/README.md:73]: This file is not there yet. Please feel free to add if you are interested

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (0xYujan) [reply docs/README.md:73]: @shrn01 but I already submit this file here #8803

Comment (shrn01) [reply docs/README.md:73]: Move this change also to #8803

Comment (0xYujan) [reply docs/README.md:73]: @shrn01 I think it's done

Comment (shrn01) [reply docs/README.md:73]: @0xYujan you can close this PR then

**Pull request closed without merging**